pkg/controllers/disruption/suite_test.go

```diff
@@ -416,6 +278,133 @@ var _ = Describe("Disruption Taints", func() {
 	})
 })
 
+/*
+	nodePoolList, err := nodepoolutil.List(ctx, kubeClient)
```

Comment:
I figure you will remove this later, but just want to callout that this is still commented out

Reply:
Thanks for reminder 👍